File: abp/src/Wallee.Boc.Vote.Application.Contracts/CandidateOrgUnits/CandidateOrgUnitCreateDto.cs
using System;

namespace Wallee.Boc.Vote.CandidateOrgUnits
{
    public class CandidateOrgUnitCreateDto
    {
        /// <summary>
        /// 组织机构号
        /// </summary>
        public Guid OrganizationUnitId { get; set; }
        /// <summary>
        /// 部门类别
        /// </summary>
        public CandidateOrgUnitCategory Category { get; set; }

        public Guid SuperiorId { get; set; }
        public string Description { get; set; } = default!;
    }
}

File: vote_vite_client/src/openapi/models/CandidateOrgUnitCreateDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { CandidateOrgUnitCategory } from './CandidateOrgUnitCategory';

export type CandidateOrgUnitCreateDto = {
    organizationUnitId?: string;
    category?: CandidateOrgUnitCategory;
    superiorId?: string;
};
